File: composeApp/src/androidMain/kotlin/co/jarias/flexapp/ui/screens/bingo/game_list/BingoGameListScreenEvents.kt
package co.jarias.flexapp.ui.screens.bingo.game_list

sealed class BingoGameListScreenEvents {
    data object OnCreateNewGameClicked : BingoGameListScreenEvents()
    data class OnPlayGame(val gameId: Long) : BingoGameListScreenEvents()
    data class OnRestartGame(val gameId: Long) : BingoGameListScreenEvents()
    data class OnDeleteGame(val gameId: Long) : BingoGameListScreenEvents()
    data object OnRetryClicked : BingoGameListScreenEvents()
}

File: iosApp/iosApp/Screens/bingo/game_list/BingoGameListScreenEvents.swift
import Foundation

enum BingoGameListScreenEvents {
    case onCreateNewGameClicked
    case onPlayGame(gameId: Int64)
    case onRestartGame(gameId: Int64)
    case onDeleteGame(gameId: Int64)
    case onRetryClicked
    case onContinueSetup(gameId: Int64)
}
